Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -227,6 +227,10 @@ - (nullable WMFContentGroup *)sectionForIndexPath:(NSIndexPath *)indexPath {
         content = [content wmf_map:^id(WMFFeedNewsStory *obj) {
             return [[obj featuredArticlePreview] articleURL] ?: [[[obj articlePreviews] firstObject] articleURL];
         }];
+    } else if ([group contentType] == WMFContentTypeOnThisDayEvent) {
+        content = [content wmf_map:^id(WMFFeedOnThisDayEvent *obj) {
+            return [[[obj articlePreviews] firstObject] articleURL];
```

Comment:
@joewalsh was a little foggy about whether this line and [L281](https://github.com/wikimedia/wikipedia-ios/pull/1545/commits/0261e7baac7d509eedfdddb711943ea797316922#diff-075df5f5584af1efc1a6c2e7138578d5R281) and [L332](https://github.com/wikimedia/wikipedia-ios/pull/1545/commits/0261e7baac7d509eedfdddb711943ea797316922#diff-075df5f5584af1efc1a6c2e7138578d5R332) were right... once things are wired up a bit more it'll probably become apparent, but thought I'd ask in the mean time.